src/ray/common/scheduling/scheduling_ids.cc

```diff
@@ -44,7 +43,7 @@ int64_t StringIdMap::Insert(const std::string &string_id, uint8_t max_id) {
   if (sit == string_to_int_.end()) {
     int64_t id = hasher_(string_id);
     if (max_id != 0) {
-      id = id % MAX_ID_TEST;
```

Comment:
Q: should this actually be 

    id = id % max_id

like in line 58?

Reply:
I feel like it's intended, so we could pass in `max_id` = 0 in tests, meanwhile still capping the max id.

Reply:
The only place that used this Insert is

https://github.com/ray-project/ray/blob/e4b9561d07b9b69e869955959886ec88d71e5efb/src/ray/raylet/scheduling/cluster_resource_scheduler_test.cc#L226

so I wonder if we can just remove this kMaxIdForTest

Reply:
Removed.